notebooks/faq/FAQs.ipynb

```diff
@@ -2266,9 +2267,6 @@
     "\n",
     "<div class=\"alert alert-block alert-info\">\n",
     "<b>VERSION NOTE:</b> Shapley value support was added with version 22.03 of the FIL backend.\n",
-    "</div>\n",
```

Comment:
Rather than eliminating this entirely, could you update with the version that CPU Shap will be added (23.04) and mention that it is currently considered experimental.